controllers/articles.controller.js

```diff
@@ -23,3 +24,15 @@ exports.getArticleById = (req, res, next) => {
 			next(err);
 		});
 };
+
+exports.patchArticle = (req, res, next) => {
+	const { article_id } = req.params;
+	const reqBody = req.body;
+	updateArticle(article_id, reqBody)
+		.then((article) => {
+			res.send(article);
```

Comment:
You should be sending this on a key of article, so it should read:res.status(200).send({article})